### PR TITLE
Community Article - Launch on Rackspace Cloud

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

File: content/cloud-servers/launch-on-rackspace-cloud.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Now it’s time to test, test, test. Does everything work as planned? What’s different than your current host? Hopefully it’s better, but if not, what needs to be changed? Test, iterate, and test some more until you’re ready to launch.
+
+### Launch day
+
+Once everything is tested, it’s time for launch. The actual switch will vary based on your current DNS host, so check with them. We encourage you to use Rackspace Cloud DNS (either as a primary or backup DNS provider), but you do not need to host your DNS with us.
+* Determine exactly what will be switched and backup your current DNS configuration. Make sure you get all components.
+* If you’re switching all of your DNS hosting, make sure your email and other systems configurations have been moved.
+
+### How to configure Rackspace DNS for Rackspace Cloud
+
```
Comment (WilliamLoy) [inline content/cloud-servers/launch-on-rackspace-cloud.md]: I would include some verbiage here that makes it clear these instructions are only intended for customer who plan to use Backspace DNS hosting. There should be a second set of instructions that explain how to compete this at their own dns host.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6205f85ab2/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-5a8bcc6313/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e178d31d5b/how-to/)

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by cyrichardson** [comment]
On it, @RackerKate!

**Comment by cyrichardson** [comment]
@RackerKate Suggestions here:
[launch-on-rackspace-cloud_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2901815/launch-on-rackspace-cloud_cyrEdits.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-fcb48c0535/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-ba428e92ff/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-37b47251fe/how-to/).

**Pull request merged**